### PR TITLE
Implemented code to make the in progress orders flow functional(#201pbe2)

### PR DESCRIPTION
**Pull request by disha1202**

### CODE REVIEW

File: src/views/InProgress.vue [hunk 1]
```diff
@@ -27,52 +27,52 @@
...
-                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
-              </ion-thumbnail>
-              <ion-label>
-                <p class="overline">WJ06-XL-PURPLE</p>
-                Juno Jacket
-                <p>Blue XL</p>
-              </ion-label>
-            </ion-item>
-          </div>
+        <div v-for="order in orders.doclist.docs" :key="order">
```
Comment (ymaheshwari1) [inline src/views/InProgress.vue:61]: We don't need an extra div for the v-for loop we can define this `v-for` condition in the div with class `order-item` and remove this extra div.

File: src/views/InProgress.vue [hunk 2]
```diff
@@ -27,52 +27,52 @@
...
           <div class="order-tags">
             <ion-chip outline>
               <ion-icon :icon="pricetagOutline" />
-              <ion-label>NN10584</ion-label>
+              <ion-label>{{ orders.doclist.docs[0].orderId }}</ion-label>
             </ion-chip>
           </div>
 
-          <div class="order-metadata">
+          <!-- <div class="order-metadata">
```
Comment (ymaheshwari1) [inline src/views/InProgress.vue:48]: Uncomment this div and instead comment the `ion-label`, as commenting this div will distort the UI.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/InProgress.vue (hunk 2 above)
Comment (disha1202) [reply src/views/InProgress.vue]: Done Sir

File: src/views/InProgress.vue (hunk 1 above)
Comment (disha1202) [reply src/views/InProgress.vue]: Done Sir

File: src/store/modules/product/actions.ts [hunk 3]
```diff
@@ -46,6 +46,51 @@ const actions: ActionTree<ProductState, RootState> = {
...
+        return filter += productId;
+      }
+    }, '');
+
+    // If there are no products skip the API call
+    if (productIdFilter === '') return;
+
+    // adding viewSize as by default we are having the viewSize of 10
+    const resp = await ProductService.fetchProducts({
+      "filters": ['productId: (' + productIdFilter + ')']
```
Comment (ymaheshwari1) [inline src/store/modules/product/actions.ts:67]: Add viewSize here which will have a value of the number of productIds.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/order/actions.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+import * as types from './mutation-types'
+
+
+const actions: ActionTree<OrderState, RootState> = {
+
+  // get in progress orders
+  async fetchInProgressOrders ({ commit }, payload) {
+    emitter.emit('presentLoader');
```
[the remark this answers is not in the document]
Comment (disha1202) [reply src/store/modules/order/actions.ts:15]: Added dismissLoader.

File: src/store/modules/order/OrderState.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export default interface OrderState {
+    inProgress: {
+      list: any,
+      total: number
```
Comment (ymaheshwari1) [inline src/store/modules/order/OrderState.ts:4]: When storing the response in status we are storing the total in the `inProgress.list.total` then why we need this total in state?

File: src/views/InProgress.vue [hunk 6]
```diff
@@ -27,31 +27,29 @@
           </ion-label>
           <ion-icon :icon="printOutline" />
         </ion-item>
-      </div> 
-
+      </div>
       <ion-button expand="block" class="desktop-only" fill="outline" @click="packOrdersAlert">{{ $t("Pack orders") }}</ion-button>
-
-      <ion-card>
+      <ion-card v-for="(orders, index) in inProgressOrders.list.inProgress" :key="index">
```
Comment (ymaheshwari1) [inline src/views/InProgress.vue:32]: The data should be returned and stored in a state in a simple manner to be used directly and we don't need to do this much nesting.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/order/OrderState.ts (hunk 5 above)
Comment (disha1202) [reply src/store/modules/order/OrderState.ts]: Updated the orders state.

**Comment by ymaheshwari1** [comment]
Closing as handled in #85

**Pull request closed without merging**